### ISSUE TITLE
Any support for windows mobile platform?

### ISSUE THREAD

**Issue by ravirupareliya** [kfiroo/react-native-cached-image#107]
Do you have any plan to give support for windows mobile app? As i can see in readme file that you have mentioned support of only Android and iOS.

**Comment by vvkaghera** [comment]
yes please share your experience in windows. which error would you face in windows mobile app development?
